js/ui/windowManager.js

```diff
@@ -571,6 +586,44 @@ const TilePreview = new Lang.Class({
     }
 });
 
+const EnableBackfaceCullingEffect = new Lang.Class({
+    Name: 'EnableBackfaceCullingEffect',
+    Extends: Clutter.Effect,
+    _init : function(params) {
+        this.parent(params);
+    },
+    vfunc_pre_paint: function() {
+        this.get_actor().set_cull_back_face(true);
+    },
+    vfunc_post_paint: function() {
+        this.get_actor().set_cull_back_face(true);
```

Comment:
Are you sure you didn't want to set this back to `false` here? Regardless, a comment on why this effect is needed at all would be nice...